### PR TITLE
Wrap by default on FlexLayout and tweak layout storybook examples

### PR DESCRIPTION
**Pull request by JoanaMMoreira**
These changes have been reviewed by Stella. They mostly include small tweaks to the storybook examples but the most important change is allowing the `FlexLayout` component to wrap by default

### CODE REVIEW

File: packages/core/src/layout/FlexLayout/FlexLayout.tsx [hunk 1]
```diff
@@ -57,7 +57,7 @@ export const FlexLayout = forwardRef<HTMLDivElement, FlexLayoutProps>(
       justify,
       separators,
       style,
-      wrap,
+      wrap = true,
```
Comment (joshwooding) [inline packages/core/src/layout/FlexLayout/FlexLayout.tsx:60]: Should this be kept as wrap or changed to disableWrap?

File: packages/core/src/layout/FlowLayout/FlowLayout.tsx [hunk 2]
```diff
@@ -1,22 +1,23 @@
-import { forwardRef, HTMLAttributes } from "react";
+import { forwardRef, HTMLAttributes, ComponentPropsWithoutRef } from "react";
 import { FlexLayout } from "../FlexLayout";
-import { FlexLayoutProps } from "../types";
+
+type FlexLayoutProps = ComponentPropsWithoutRef<typeof FlexLayout>;
```
Comment (joshwooding) [inline packages/core/src/layout/FlowLayout/FlowLayout.tsx:4]: Can we use FlexLayoutProps from FlexLayout?

File: packages/core/src/layout/StackLayout/StackLayout.tsx [hunk 3]
```diff
@@ -1,18 +1,19 @@
-import { forwardRef, HTMLAttributes } from "react";
+import { forwardRef, HTMLAttributes, ComponentPropsWithoutRef } from "react";
 import { FlexLayout } from "../FlexLayout";
-import { FlexLayoutProps } from "../types";
+
+type FlexLayoutProps = ComponentPropsWithoutRef<typeof FlexLayout>;
```
Comment (joshwooding) [inline packages/core/src/layout/StackLayout/StackLayout.tsx:4]: Same here?

**Review by joshwooding — CHANGES_REQUESTED** [review]
See above + the conflicts please :)

File: packages/core/src/layout/FlexLayout/FlexLayout.tsx (hunk 1 above)
Comment (JoanaMMoreira) [reply packages/core/src/layout/FlexLayout/FlexLayout.tsx]: I think `disableWrap` makes sense, I've amended the prop

**Review by joshwooding — APPROVED** [review]

**Review by joshwooding — APPROVED** [review]

**Comment by origami-z** [comment]
Why was the `wrap` being set by default? This shouldn't be an accessibility decision? We changed to `nowrap` previously because that is the default value for `flex-wrap`. This now causes mind bending debug issue when using flex layout...?

**Comment by JoanaMMoreira** [comment]
> Why was the `wrap` being set by default? This shouldn't be an accessibility decision? We changed to `nowrap` previously because that is the default value for `flex-wrap`. This now causes mind bending debug issue when using flex layout...?

Stella's point was around some of the examples having a double scroll bar when zoomed in so her suggestion was that the component should always wrap by default. If we think this causes debugging issues maybe we could revert the change and add an accessibility section to the docs that explains this?

**Comment by origami-z** [comment]
We can make the examples wrap, but i don't think think changing default value of any CSS is a good idea. Personally, i'm expecting flex and other layouts are some drop in replacement for not writing CSS line by line myself, if we enforce different default value than what CSS gives, we're driving into corners.

**Comment by JoanaMMoreira** [comment]
> We can make the examples wrap, but i don't think think changing default value of any CSS is a good idea. Personally, i'm expecting flex and other layouts are some drop in replacement for not writing CSS line by line myself, if we enforce different default value than what CSS gives, we're driving into corners.

Makes sense although I wonder if changing the examples only (particularly the "default" one) would be a bit misleading as well

**Pull request merged**